src/main/java/ca/ciccc/typinggame/controller/TypingGameController.java

```diff
@@ -0,0 +1,32 @@
+package ca.ciccc.typinggame.controller;
+
+import ca.ciccc.typinggame.model.Word;
+import javafx.scene.Scene;
+
+import java.util.Deque;
```

Comment:
Unfortunately, the google-java-format IntelliJ plugin doesn't fix the import order
https://github.com/google/google-java-format#intellij-android-studio-and-other-jetbrains-ides

There should be the other way, but you might have to fix it manually